test/integration/test_conferences.py

```diff
@@ -0,0 +1,488 @@
+                'os': OPERATING_SYSTEM,
+                'language': 'python' + PYTHON_VERSION,
+                'type': 'CONFERENCE'
+            }
+        )
+
+        # Get the test id from the response
+        test_id = json.loads(response.data)
+
+        # Make a CreateCall body and assign the appropriate params
+
+        call_body = CreateCall(to=MANTECA_IDLE_NUMBER, _from=MANTECA_ACTIVE_NUMBER, application_id=MANTECA_APPLICATION_ID, answer_url=answer_url, tag=test_id)
+
+        # Make the call
+        create_call_response: CreateCallResponse = self.calls_api_instance.create_call(BW_ACCOUNT_ID, call_body)
+
+        # Verify info about the call
+        assert_that(create_call_response, has_properties(
+            "call_id", instance_of(str),
+            'account_id', BW_ACCOUNT_ID,
+            'application_id', MANTECA_APPLICATION_ID,
+            'to', MANTECA_IDLE_NUMBER,
+            '_from', MANTECA_ACTIVE_NUMBER
+        ))
+
+        # Getting our ConferenceId and returning the test_id from Manteca and the callId
+
+        list_conferences_response = self.conference_api_instance.list_conferences(BW_ACCOUNT_ID, _return_http_data_only=False)
+    
+        time.sleep(self.TEST_SLEEP)
+        assert_that(list_conferences_response[1], 200)
+        conferenceId = (list_conferences_response[0][len(list_conferences_response[0])-1].id)
+
+
+        get_conference_response = self.conference_api_instance.get_conference(BW_ACCOUNT_ID, conferenceId, _return_http_data_only=False)
+        assert_that(get_conference_response[1], 200)       
+   
+        self.callIdArray.append(create_call_response.call_id)
+
+        return (test_id, create_call_response.call_id)
+
+    def validate_recording(self, recording: ConferenceRecordingMetadata, conference_id: str) -> None:
+        """
+        Validate the given recording metadata.
+        Args:
+            recording (ConferenceRecordingMetadata): The recording metadata to validate.
+            conference_id (str): The call id associated with the given recording.
+        """
+        assert_that(recording.status,'complete')
+        assert_that(recording.file_format,FileFormatEnum('wav'))
+
+    def test_conference_and_members(self):
+        """
+        Tests a successful flow of creating and ending a conference.
+        The following endpoints are tested in this flow:
+            - list_conferences
+            - get_conference_information
+            - get_conference_member
+            - update_conference_member
+            - update_conference_
+            - update_conference_bxml
+        """
+
+        # Create the call
+        answer_url = MANTECA_BASE_URL + "/bxml/joinConferencePause"
+        (test_id, call_id) = self.create_conference(answer_url)
+
+        # List Conferences
+        list_conferences_response = self.conference_api_instance.list_conferences(BW_ACCOUNT_ID, _return_http_data_only=False)
+
+        assert_that(list_conferences_response[1], 200)
```

Comment:
More thinking out loud - I _desipise_ the fact that `_return_http_data_only` returns a tuple and not a [named tuple](https://www.geeksforgeeks.org/namedtuple-in-python/) - if we can implement this without breaking anything (which I think we can since named tuples allow you to reference the key) - it might be a nice addition. Calling the index feels very cheesy to me